bulletroboy/operator.py

```diff
@@ -22,8 +22,14 @@ def __init__(self, body_id):
 		self.links = self.get_links()
 		self.movements = Movements(self)
 		self.ef_publisher = self.create_publisher(PoseStamped, '/roboy/simulation/operator/pose/endeffector', 10)
+		self.srv = self.create_service(HumanLinkIdFromName, 'human_link_id_from_link_name', self.human_link_id_from_link_name)
 		
 
+	def human_link_id_from_link_name(self, request, response):
```

Comment:
so far people seem to use a verb in the function names, like `get_link_id_from_name()`. for me personally it makes the code more understandable, it's not a requirement of course, just a suggestion